File: FizzBuzz/fizzbuzz.py
def ch(x):
  def p(n):
    return (x % n == 0)
  return ('FizzBuzz' if p(15) else 'Fizz' if p(3) else 'Buzz' if p(5) else x)

print(list(map(ch, range(1,101))))

File: FizzBuzz/fizzbuzz.c
#include <stdio.h>

int main(void)
{
  for (int n = 1; n <= 100; n++)
    if (n % 3 == 0 && n % 5 == 0) printf("FizzBuzz ");
    else if (n % 3 == 0) printf("Fizz ");
    else if (n % 5 == 0) printf("Buzz ");
    else printf("%d ", n);
  return (0);
}
